Source/gmenu.cpp

```diff
@@ -423,7 +423,7 @@ void gmenu_enable(TMenuItem *pMenuItem, BOOL enable)
 	if (enable)
 		pMenuItem->dwFlags |= 0x80000000;
 	else
-		pMenuItem->dwFlags &= 0x7F000000;
+		pMenuItem->dwFlags &= 0x7FFFFFFF;

```

Comment:
This was decompiled incorrectly before?

Reply:
It should be `pMenuItem->dwFlags &= ~0x80000000;`. Compiler will optimize it down to `[eax+3], 80h`

Reply:
@mewmew yes, I wanted to make a comment about it but lost track of what PR it was in :) Basically the same issues as with the BoyItem.

@galaxyhaxz Thanks, corrected it.